### PR TITLE
feature: add docker-compose for local development

### PR DESCRIPTION
**Pull request by prakashdivyy**

### CODE REVIEW

**Comment by mgilangjanuar** [comment]
Thank you so much! 🎊 

Btw, can you update the Readme for how to build with Docker? And what does BUILD_CONTEXT mean? 🤔 
image

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        BUILD_CONTEXT: server
+        NPM_TOKEN: ${NPM_TOKEN}
+    environment:
+      DB_NAME: postgres
+      DB_HOST: db
+      DB_USERNAME: ${DB_USERNAME}
+      DB_PASSWORD: ${DB_PASSWORD}
+    depends_on:
+      - db
+  cache:
```
Comment (mgilangjanuar) [inline docker/docker-compose.yml:83]:
```suggestion
  redis:
```

File: docker/.env.example [hunk 2]
```diff
@@ -0,0 +1,27 @@
+NPM_TOKEN=your_github_token
+
+# server configuration
+RPS=
+
+TG_API_ID=
+TG_API_HASH=
+TG_BOT_TOKEN=
+TG_BOT_OWNER_ID=
+
```
Comment (mgilangjanuar) [inline docker/.env.example:10]:
```suggestion
TG_BOT_ERROR_REPORT_ID=

```

**Comment by JamesDAdams** [comment]
I try to start the docker-compose 
image

**Comment by mgilangjanuar** [comment]
@JamesDAdams  Oh you should install `react-scripts` globally first with: `npm I react-scripts -g`

**Pull request merged**